db/schema.rb

```diff
@@ -111,6 +111,7 @@
     t.integer "aid_applications_count", default: 0, null: false
     t.integer "total_payment_cards_count", default: 0, null: false
     t.string "county_names", default: [], array: true
+    t.string "phone_number"
```

Comment:
It looks like one org in the spreadsheet has 2 numbers. Could we make this more of a free text field like "contact information" and expect the person inputting the data to format it?

Reply:
Good call! Thanks, I'll do that.